keras_cv/layers/preprocessing/invert.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import tensorflow as tf
+
+
+class Invert(tf.keras.layers.Layer):
+    """Inverts the image pixels.
+
+    Usage:
+        ```
+        invert = Invert()
+        (images, labels), _ = tf.keras.dataset.cifar10.load_data()
+        # Note that images are Tensor with values in the range [0, 255]
+        images = invert(images)
+        ```
+
+    Call arguments:
+        images: Tensor of type int or float, pixels in range [0, 255].
+    """
+
+    def call(self, images):
+        return 255 - images
```

Comment:
What's the expected behavior if the values are outside of this range.  Should we Clip the Values to 0-255?
Also, can the same layer be used when its scaled to 0-1 instead of 0-255?

This layer makes an assumption that all values are 0-255.  Can this be parameterized?

Reply:
We should raise a `ValueError` when a value is outside of the range - as an `Invert` function should only have a domain of [minpixel, maxpixel]

Reply:
1. I think we should have a init param to control what's the range of the value we should accept (eg, if its on the scale of 0~1 or 0~255). Based on imagenet and most of the other datasets, I think we should default the value to 255
2. Raise ValueError means we have to check against the tensor value of the input, which might not be a cheap computation and we probably would like to not pay this every single time.

Reply:
Very good point on 2.) - thanks for pointing that out.

Reply:
1) Agreed on `min_value` and `max_value` parameters. I will be adding this.

2) Is it decided not to include input values validation in `call` method? It is not only costly, but also problematic in graph mode. 

Personally, I think it could be user's responsibility to test if they are passing values of correct range. To highlight this we could not provide default values of 0 and 255 to `min_value` and `max_value` and require the user to explicitly provide them. 

Please, let me know what you think.

Reply:
@sampathweb I don't think that clipping values is going to help much. I would not add it here.

Example scenario 1: Invert with values 0 and 1 but user passes 0-255.
The values would be clipped to 0 and 1. These would be the only 2 unique values in the image.

Example scenario 2: Invert with values 0-255 but user passes 0-1
Many values close to 0 and 255, clipping does not help here and image is almost unchanged.

EIther way the layer does not do what it's supposed to. I would still transfer the responsibility of validating input to the end user.